src/components/Camera/helpers/visionCameraWrapper.e2e-mock

```diff
@@ -0,0 +1,19 @@
+// This wraps the react-native-vision-camera component and methods we use,
+// so we can mock them for e2e tests in simulators without camera.
+import { useFrameProcessor } from "react-native-vision-camera";
```

Comment:
Note that we are not mocking the useFrameProcessor hook. So, in the e2e test a real frame processor in the sense of react-native-vision-camera is built. As you can see in the next wrapper file our plugin is not used though in this frame processor and only a mocked prediction is immediately returned.